rust-runtime/inlineable/Cargo.toml

```diff
@@ -32,7 +32,7 @@ once_cell = "1.16.0"
 percent-encoding = "2.2.0"
 pin-project-lite = "0.2"
 regex-lite = "0.1.5"
-url = "2.2.2"
+url = "2.5.0"
```

Comment:
Do we need to update version number [here](https://github.com/smithy-lang/smithy-rs/blob/2392a24ed611447fb4789d93873bf19e9e2a8e83/codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/CargoDependency.kt#L249) as well?

Reply:
What's the reason for upgrading it at all?

Reply:
To be in sync with what we specify in `inlineable/Cargo.toml`?